docs/helm/endpoints_api.md

```diff
@@ -0,0 +1,323 @@
+**List Helm Releases**
+----
+  _Returns List installed helm charts in JSON_
+
+* **URL**
+
+     `/api/helm/releases`
+
+* **Method:**
+
+  `GET`
+
+*  **URL Params**
+
+   `ns=[string]` - Namespace
+
+* **Success Response:**
+
+  * **Code:** 200 <br />
+
+* **Error Response:**
+
+  * **Code:** 400 BAD REQUEST <br />
+    **Content:** `{ error : "error message" }`
+
+
+**Get Helm Release**
+----
+  _Returns installed helm release in JSON_
+
+* **URL**
+
+     `/api/helm/release`
+
+* **Method:**
+
+  `GET`
+
+*  **URL Params**
+
+   `ns=[string]` - Namespace
+
+   `name=[string]` - Helm Release Name
+
+* **Success Response:**
+
+  * **Code:** 200 <br />
+  * JSON encoded [Release structure](https://github.com/helm/helm/blob/master/pkg/release/release.go#L22)
+
+* **Error Response:**
+
+  * **Code:** 400 BAD REQUEST <br />
+    **Content:** `{ error : "error message" }`
+
+
+
+**Get Helm Release History**
+----
+  _Returns installed helm release history in JSON_
+
+* **URL**
+
+     `/api/helm/release/history`
+
+* **Method:**
+
+  `GET`
+
+*  **URL Params**
+
+   `ns=[string]` - Namespace
+
+   `name=[string]` - Helm Release Name
+
+* **Success Response:**
+
+  * **Code:** 200 <br />
+  * JSON encoded array of [Release structure](https://github.com/helm/helm/blob/master/pkg/release/release.go#L22)
+
+* **Error Response:**
+
+  * **Code:** 502 BAD GATEWAY <br />
```

Comment:
This doesn't seems right, since `502` RC should be used in case of server error response code indicates that the server, while acting as a gateway or proxy, received an invalid response from the upstream server.
`400` and `404` should be used here.

Reply:
might be, but that is not in the focus of PR - we have just extracted the existing doc into a separate file.